### DISCUSSION TITLE
Download Queue Headings Spacing Increase?

### DISCUSSION THREAD

**Discussion by SOLOSAIR** [Ascoware/get-iplayer-automator#344] [Ideas]
Does anyone think that now the app window size can be increased as desired, the default width of the headings spaces could/should be increased?  It is only a minor consideration I know, but having to manually increase the column headings to the right every time I use the app - especially the 'Show' and 'Episode' sections considerably -  to enable seeing full information, I find is somewhat tiresome.

**Comment by skovatch** [comment]
I can widen the columns. Storing those column widths is surprisingly annoying, and not supported in AppKit.

**Reply by SOLOSAIR** [reply]
Sorry, I'm not a developer, just a viewer, so does this mean it is possible for you you do it but apple prevents a permanent change so it cannot be done?

**Reply by skovatch** [reply]
No, it can be done. It's just that (IMO) storing the widths of table view columns in some way so they can be restored is a pretty basic feature that should have been provided by Apple but wasn't. I have read some articles by other devs that came up with a solution but haven't put in a lot of effort into trying them out.

**Comment by SOLOSAIR** [comment]
ah good!    Thanks for explaining. Many thanks for all you do for us users.    Not having to expand all the headings at every launch would be a better user experience for all of us. 🙂👍🏻
  
  
  
 Sarah
  
  
  

  
  
>

**Comment by SOLOSAIR** [comment]
Works a treat!  So much better!  Many, many thanks! 😃👍🏻
